catalog/admin/admin-poller-service/pom.xml

```diff
@@ -59,6 +59,30 @@
             <groupId>ddf.action.core</groupId>
             <artifactId>action-core-api</artifactId>
         </dependency>
+        <dependency>
```

Comment:
This didn't require the gmavenplus plugin? Is it coming out of a pom farther up in the hierarchy?